src/beamlime/executables/options.py

```diff
@@ -70,5 +70,11 @@ def visualization_arg_parser(
         help="Set logging level. Default is INFO.",
         type=str,
     )
+    group.add_argument(
+        "--workflow-name",
+        default="dummy",
+        help="Name of the workflow that produces the visualization",
+        type=str,
+    )
```

Comment:
I think plugins should be able to add it's own argument group in to the parser.

For example, ``beamlime`` expects plugins to have a function that adds argument group, sth like... ``workflow_arg_parser(parser: argparse.ArgumentParser)`` so that it adds its own argument group into the parser.
So that ``beamlime`` doesn't have to know about the arguments that only those plugins use.

Or did you want to add an argument for the name of the "workflow plugin", like ``esssans``...?

Reply:
> I think plugins should be able to add it's own argument group in to the parser.

We should not do this for now. Configuration handling is not part of this milestone, and neither of the next (but the one after that).

Reply:
Yeah what I was trying to say was,
**If** it is done, it should be done by the plugins.